frontend/src/components/auth/LoginForm.vue

```diff
@@ -101,23 +109,55 @@
             }),
             onLogin() {
                 if (this.$refs.form.validate()) {
+                    this.isLoading = true;
                     this.login({
                         email: this.email,
                         password: this.password
                     }).then(() => {
+                        this.isLoading = false;
```

Comment:
no need to duplicate same action twice, just add `then` after catch statement or `finally`, and put this prop change there